articlewebpage.cpp

```diff
@@ -1,9 +1,15 @@
 #include "articlewebpage.h"
 #include "utils.hh"
+#include "globalbroadcaster.h"
+#include <QColor>
 
 ArticleWebPage::ArticleWebPage(QObject *parent)
   : QWebEnginePage{parent}
 {
+  // when opening a new tab in dark reader mode, there will be a white flash without this
+  if( GlobalBroadcaster::instance()->getPreference()->darkReaderMode ) {
+    setBackgroundColor( Qt::black );
+  }
```

Comment:
`setBackgroundColor(Qt::Transparent)` should be better

Reply:
It cannot be transparent, because in light UI, there will be flash when opening new tabs. There is another code that do the same.

Reply:
I think the flash come from the ArticleView.   when WebEngineView is hidden ,
the background frame is ArticleView.

if ArticleView has a different color ,  a flash will be happening

Reply:
I changed the ArticleView's background color to match the Article's background in Darkreader mode.